pages/blog/index.js

```diff
@@ -129,18 +123,30 @@ function BlogIndex({posts}) {
 }
 
 BlogIndex.defaultProps = {
-  posts: null
+  posts: null,
+  pagination: null,
+  tags: null,
+  tagsList: null
 }
 
 BlogIndex.propTypes = {
-  posts: PropTypes.array
+  posts: PropTypes.array,
+  pagination: PropTypes.object,
+  tags: PropTypes.array,
+  tagsList: PropTypes.array
 }
 
-export async function getServerSideProps() {
-  const posts = await getPosts()
+export async function getServerSideProps(ctx) {
```

Comment:
Seul `query` est utilisé, on peut donc déstructurer `ctx`
```suggestion
export async function getServerSideProps({query}) {
```